calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp

```diff
@@ -119,19 +132,21 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
     const std::vector<std::pair<Eigen::Vector3d, Eigen::Vector3d>> & matches,
     builtin_interfaces::msg::Time & time);
 
-  std::tuple<pcl::PointCloud<PointType>::Ptr, pcl::PointCloud<PointType>::Ptr, double, double>
-  getPointsSetAndDelta();
+  std::tuple<pcl::PointCloud<PointType>::Ptr, pcl::PointCloud<PointType>::Ptr> getPointsSet();
+  std::tuple<double, double> getDelta(std::vector<Track> converged_tracks, bool is_crossval);
```

Comment:
Please use a more descriptive name
getRotationDelta
get2DRotationDelta
etc

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/0f648c0f7e2ef653b4c604e4155bc7d36cc7c093